app/models/user.rb

```diff
@@ -27,6 +27,11 @@ def lending_rights?(item)
     false
   end
 
+  def items
+    # get all items where the user is a manager of any group
+    Item.joins(:manager_groups).where(groups: { id: groups })
```

Comment:
- structure (optional): if possible, it might be preferable to use has_many relations for clarity